core/src/test/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -383,6 +383,11 @@ public static String randomRealisticUnicodeOfCodepointLength(int codePoints) {
         }
         return array;
     }
+    
+    public static String randomTimeValue() {
+        final String[] values = new String[]{"d", "H", "ms", "s", "S", "w"};
+        return randomIntBetween(0, 3) + randomFrom(values);
```

Comment:
no I meant that I don't understand why we randomize between 0 and 3. Can you elaborate?